### PR TITLE
Standard Transformer UI

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
**Does this PR introduce a user-facing change?**:

```release-note
- Update model deployment page to include the settings of standard transformer deployment
- Update model details page to include standard transformer details
```

**Screenshots**

- Model deployment page
  image

- Model version details page
  Screen Shot 2021-02-02 at 2 49 08 PM

### CODE REVIEW

File: ui/src/pages/version/Version.js [hunk 1]
```diff
@@ -204,7 +204,27 @@ const Version = ({ projectId, modelId, versionId, endpointId, ...props }) => {
...
+                        memory_request: "500Mi",
+                        min_replica: 1,
+                        max_replica: 4
+                      },
+                      transformer_type: "standard",
+                      env_vars: [
+                        {
+                          name: "TRANSFORMER_CONFIG",
+                          value:
+                            '{"transformerConfig":{"feast":[{"project":"project_1","entities":[{"name":"user_id_1","valueType":"STRING","jsonPath":"user_id_1"}],"features":[{"name":"feast_test_metrics:string_feature","valueType":"STRING","defaultValue":"1"}]},{"project":"project_2","entities":[{"name":"user_id_2","valueType":"STRING","jsonPath":"user_id_2"}],"features":[{"name":"feast_test_metrics:string_feature","valueType":"STRING","defaultValue":"2"}]}]}}'
```
Comment (pradithya) [inline ui/src/pages/version/Version.js:223]: What is it for?

Comment (ariefrahmansyah) [reply ui/src/pages/version/Version.js]: This is for testing. I'll clean this

File: ui/src/config.js [hunk 2]
```diff
@@ -53,6 +53,7 @@ const config = {
   TIMEOUT: 20000,
   MERLIN_API: getEnv("REACT_APP_MERLIN_API"),
   MLP_API: getEnv("REACT_APP_MLP_API"),
+  FEAST_API: getEnv("REACT_APP_FEAST_API"),
```
Comment (pradithya) [inline ui/src/config.js:56]: Maybe make it clearer `FEAST_CORE_API` ?

File: ui/src/version/deployment/components/EndpointDeployment.js [hunk 3]
```diff
@@ -111,6 +114,9 @@ export const EndpointDeployment = ({
   }, [actionTitle, breadcrumbs]);
 
   const [request, setRequest] = useState({});
+  // useEffect(() => {
```
Comment (pradithya) [inline ui/src/version/deployment/components/EndpointDeployment.js:117]: Remove dead code

File: ui/src/version/deployment/components/EndpointDeployment.js [hunk 4]
```diff
@@ -262,6 +268,12 @@ export const EndpointDeployment = ({
                     request.transformer || {
                       image: "",
                       resource_request: defaultResourceRequest
+                      // enabled: true,
```
Comment (pradithya) [inline ui/src/version/deployment/components/EndpointDeployment.js]: Dead code

**Comment by pradithya** [comment]
Question:
- How user configure cpu / memory request from UI?
- I think in sdk user is able to configure those value, will it be override if user redeploy using UI?

**Comment by ariefrahmansyah** [comment]
> Question:
> 
> * How can user configure cpu / memory request from UI?

I thought that we were going to hide the complexity of configuring standard transformer resources from the UI and just use the environment target's default resources configuration. If you think user need it in the UI, I think we can add an accordion component that could display and hide the resources config.

> * I think in sdk user is able to configure those value, will it be override if user redeploy using UI?

Yes. Both of them can override each other config.

File: ui/src/version/deployment/components/StandardTransformerForm.js [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+  const [configInitialized, setConfigInitialized] = useState(false);
+  useEffect(() => {
+    if (configInitialized) {
+      return;
+    }
+
+    if (transformer.env_vars && transformer.env_vars.length > 0) {
+      const envVar = transformer.env_vars.find(
+        e => e.name === "TRANSFORMER_CONFIG"
```
Comment (pradithya) [inline ui/src/version/deployment/components/StandardTransformerForm.js:49]: Can you make `TRANSFORMER_CONFIG` a constant? Also the value should be `STANDARD_TRANSFORMER_CONFIG`

**Review by pradithya — APPROVED** [review]
LGTM

**Pull request merged**